spec/test_app/app/lib/random_filename_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe RandomFilename do
+  before { allow(SecureRandom).to receive(:uuid).and_return('uuid') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.